tests/containers/host_configuration.pm

```diff
@@ -27,8 +27,18 @@ sub run {
     # Update the system to get the latest released state of the hosts.
     # Check routing table is well configured
     if ($host_distri =~ /sles|opensuse/) {
+        my $host_version = (get_required_var('HOST_VERSION') =~ /SP/) ? "SLES_" . (get_required_var('HOST_VERSION') =~ s/-SP/_SP/r) : get_required_var('HOST_VERSION');
```

Comment:
```suggestion
        my $host_version = (get_required_var('HOST_VERSION') =~ /SP/) ? "SLE_" . (get_required_var('HOST_VERSION') =~ s/-SP/_SP/r) : get_required_var('HOST_VERSION');
```

At least if I look at http://download.suse.de/ibs/SUSE:/CA, it should be `SLE_` not `SLES_`.

Reply:
Hmmm I'm confused because it's working now with SLES...
I'll change it